packages/near-api-js/src/account_multisig.ts

```diff
@@ -329,17 +330,23 @@ export class Account2FA extends AccountMultisig {
         ] : [];
     }
 
-    async get2faDisableKeyConversionActions() {
-        const { accountId } = this;
-        const accessKeys = await this.getAccessKeys();
-        const lak2fak = accessKeys
-            .filter(({ access_key }) => access_key.permission !== 'FullAccess')
+    async get2faLimitedAccessKeys() {
+        return (await this.getAccessKeys())
             .filter(({ access_key }) => {
+                if (access_key.permission === 'FullAccess') {
```

Comment:
Small change when I factored this out, seems excessive to do sequential filters so I consolidated them